test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1

```diff
@@ -0,0 +1,367 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Experimental Feature Basic Tests - Feature-Disabled" -tags "CI" {
+
+    BeforeAll {
+        $skipTest = $EnabledExperimentalFeatures.Contains('ExpTest.FeatureOne')
+
+        if ($skipTest) {
+            Write-Verbose "Test Suite Skipped. The test suite requires the experimental feature 'ExpTest.FeatureOne' to be disabled." -Verbose
+            $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+            $PSDefaultParameterValues["it:skip"] = $true
+        } else {
+            $TestModule = Join-Path $PSScriptRoot "assets" "ExpTest"
+            $AssemblyPath = Join-Path $TestModule "ExpTest.dll"
+            if (-not (Test-Path $AssemblyPath)) {
+                $SourcePath = Join-Path $TestModule "ExpTest.cs"
+                Add-Type -Path $SourcePath -OutputType Library -OutputAssembly $AssemblyPath
+            }
+            $moduleInfo = Import-Module $TestModule -PassThru
+        }
+    }
+
+    AfterAll {
+        if ($skipTest) {
+            $global:PSDefaultParameterValues = $originalDefaultParameterValues
+        } else {
+            Remove-Module -ModuleInfo $moduleInfo -Force -ErrorAction SilentlyContinue
+        }
+    }
+
+    It "No experimental features is enabled" {
```

Comment:
should be either `features are` or `feature is`

Reply:
Fixed.